indexer/packages/postgres/src/types/query-types.ts

```diff
@@ -100,6 +101,11 @@ export interface SubaccountQueryConfig extends QueryConfig {
   [QueryableField.UPDATED_ON_OR_AFTER]?: string;
 }
 
+export interface SubaccountUsernamesQueryConfig extends QueryConfig {
+  [QueryableField.USERNAME]?: string;
+  [QueryableField.SUBACCOUNT_ID]?: string;
```

Comment:
What's the purpose of findall if the query only takes a singular subaccountId and or a single username? I think these should both be lists of strings otherwise you should use findBySubaccountId or findByUsername

Reply:
Findall was basically a substitute for a couple functions(findBySubaccountId and just getting all the rows). I dont think there is an active case right now where we require lists though